src/assets/translations/en/main.json

```diff
@@ -155,6 +155,7 @@
     "symmetric": "Symmetric",
     "asymmetric": "Asymmetric (%{assetSymbol})",
     "import": "Import",
+    "accountsLoading": "Accounts are currently loading",
```

Comment:
`Loading Accounts` is referenced on the issue and more concise for button text imo